style.css

```diff
@@ -0,0 +1,241 @@
+/* QUESTIONS TO ASK:  */
+/* 1. why is my heading section tagline and call to action not aligned?*/
+
+/* overall style */
+div {
+    /* background-color: navy; */
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+ul {
+    padding: 0px;
+}
+
+/* nativation bar style */
+nav {
+    background-color: pink;
+    height: 3em;
+    font-weight: bolder;
+    padding-top: 10px;
+}
+nav ul {
+    /* border: solid; */
+    list-style: none;
+    display: grid;
+    grid-template: 25% 75% / 70% 15% 15%;
+    /* grid-template-columns: 70% 15% 15%; */
+    /* grid-template-rows: 25% 75%; */
+}
+nav ul li:first-child{
+    /* border: solid; */
+    grid-column-start: 1;
+    grid-row-start: 2;
+}
+nav ul li:nth-of-type(2) {
+    /* border: solid; */
+    grid-column-start: 2;
+    grid-row-start: 2;
+}
+nav ul li:nth-of-type(3) {
+    /* border: solid; */
+    grid-column-start: 3;
+    grid-row-start: 2;
+}
+
+/* heading style */
+#heading {
+    border-bottom: solid 1px grey;
+    background-color: lightgrey;
+    height: 20em;
+    display: grid;
+    grid-template: 1fr 1fr .5fr 1fr / 1fr 1fr 1fr;
+    /* flex-direction: column;
+    align-items: center; */
+    /* justify-content: center; */
+}
+#heading h1 {
+    /* border: solid; */
+    grid-column-start: 2;
+    grid-row-start: 2;
+    text-align: center;
+    font-weight: bold;
+}
+/* TODO: why is text not centered in border box */
+#heading sub {
+    /* border: solid; */
+    margin-bottom: 15px;
+    grid-column-start: 2;
+    grid-row-start: 3;
+    text-align: center;
+    font-weight: bold;
+    font-size: 1.9vw;
+    overflow-wrap: break-word;
+    position: static; 
+    line-height: normal;
+}
+#heading a {
+    border: solid;
+    margin: auto;
+    border-radius: 5px;
+    padding: 10px;
+    /* height: 50%; */
+    grid-column-start: 2;
+    grid-row-start: 4;
+    text-align: center;
+    /* font-size: 1px; */
+    color: grey;
+    text-decoration: none;
+}
+#heading a:hover {
+    color: yellow;
+}
+
+/* QUOTE styling (2) */
+.quote {
+    border-bottom: solid 1px grey;
+    background-color: antiquewhite;
+    height: 15em;
+    display: grid;
+    grid-template: 1fr 1fr 1fr 1fr / 1fr 5fr 1.5fr .5fr;
+    font-style: italic;
+}
+.quote blockquote {
+    /* border: solid; */
+    line-height: 200%;
+    grid-column-start: 2;
+    margin-left: 20px;
+    margin-right: 20px;
+    grid-row-start: 2;
+}
+.quote p::before {
+    content: "--";
+}
+.quote p {
+    /* border: solid; */
+    grid-column-start: 3;
+    grid-row-start: 3;
+}
+
+/* WHY styling */
+#why {
+    /* background-color: lavender; */
+    height: 20em;
+    border-bottom: solid 1px;
+}
+#why h2, #team h2, #sponsors h2{
+    /* border: solid; */
+    text-align: center;
+    width: 100%;
+    padding-top: 3%;
+    padding-bottom: 3%;
+}
```

Comment:
I like how your selectors are very specific which might be great if html changes in the future. However, if you are applying the same style to all h2 elements, it might be good to just target h2 element.